app/models/course_option.rb

```diff
@@ -11,6 +11,13 @@ class CourseOption < ApplicationRecord
     part_time: 'part_time',
   }
 
+  enum vacancy_status: {
+    both_full_time_and_part_time_vacancies: 'B',
+    part_time_vacancies: 'P',
+    full_time_vacancies: 'F',
+    no_vacancies: '',
```

Comment:
I wonder if we should persist the string human readable strings like `full_time_vacancies`, instead of "B/P/F", to avoid coupling to Find?

Reply:
Agreed, clearer when inspecting records in the DB.